CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This file is influenced by http://keepachangelog.com/.
 - Include git tag and sha1 on editorial footer
 
 ### Changed
+- Truncated SHA on editorial page and improved clarity around version
```

Comment:
This is probably not needed because it's be covered by the previous changelog update that is in the same version.